src/parser.yy

```diff
@@ -97,6 +97,7 @@ void yyerror(bpftrace::Driver &driver, const char *s);
 %token <std::string> MAP "map"
 %token <std::string> VAR "variable"
 %token <long> INT "integer"
+%token <long> CINT "colon surrounded integer"
```

Comment:
I think is pretty nasty but I can't think of anything better. I had to do this because `path` sucks up just too wide of a character range.